test/browser/utils.tsx

```diff
@@ -2,7 +2,7 @@ import { act, render, screen } from '@testing-library/react';
 import { page, userEvent } from '@vitest/browser/context';
 import { css } from '@linaria/core';
 
-import DataGrid from '../../src';
+import { DataGrid } from '../../src';
 import type { DataGridProps } from '../../src';
```

Comment:
Can combine these

Reply:
Some places are combined, some not, dunno if we want to prefer one over the other. 🤔 
Should be handled by prettier or something.